Casks/w/windsurf-next.rb

```diff
@@ -0,0 +1,43 @@
+cask "windsurf-next" do
```

Comment:
```suggestion
cask "windsurf@next" do
```

The filename needs to be updated as well.

Reply:
it's also done